src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -76,4 +79,38 @@ public void MercuryCuratedNavigationTests_002_navigateThroughSection() {
 
     UrlChecker.isPathContainedInCurrentUrl(driver, ROOT_PATH_SECTION + section.getTitle());
   }
+
+  // CCT11
+  @Test(groups = {"MercuryCuratedNavigationTests_003", "MercuryCuratedNavigationTests",
+                  "MercuryCuratedContentTests", "Mercury"})
+  public void MercuryCuratedNavigationTests_003_navigateThroughNamespaces() {
+    CuratedContentPageObject category = new CuratedContentPageObject(driver);
+    category
+        .openCuratedContentPage(wikiURL, MercurySubpages.CC_CATEGORY_ARTICLES)
+        .isArticleIconVisible()
+        .clickOnCuratedContentElementByIndex(0)
+        .waitForLoadingSpinnerToFinishReloadingPage();
+    UrlChecker.isPathContainedInCurrentUrl(driver, ROOT_ARTICLE_PATH);
+
+    category
+        .openCuratedContentPage(wikiURL, MercurySubpages.CC_CATEGORY_BLOGS)
+        .isBlogIconVisible()
+        .clickOnCuratedContentElementByIndex(0)
+        .waitForLoadingSpinnerToFinishReloadingPage();
+    UrlChecker.isPathContainedInCurrentUrl(driver, ROOT_ARTICLE_PATH);
+
+    category
+        .openCuratedContentPage(wikiURL, MercurySubpages.CC_CATEGORY_BLOGS)
+        .isImageIconVisible()
+        .clickOnCuratedContentElementByIndex(0)
+        .waitForLoadingSpinnerToFinishReloadingPage();
+    UrlChecker.isPathContainedInCurrentUrl(driver, ROOT_ARTICLE_PATH);
+
+    category
+        .openCuratedContentPage(wikiURL, MercurySubpages.CC_CATEGORY_BLOGS)
+        .isVideoIconVisible()
+        .clickOnCuratedContentElementByIndex(0)
+        .waitForLoadingSpinnerToFinishReloadingPage();
```

Comment:
this page should be called "waitForPageToLoad()" - base on this name you can get an impression that spinner is doing the reloading.